satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -791,10 +848,11 @@ datasets:
   # ---- CSR products ------------
   nir39:
     name: nir39
+    long_name: TOA Brightness Temperature at 3.9um
```

Comment:
Again I suggest to indicate in the filename that this is a segment mean value of clear pixels --> "TOA Brightness Temperature at 3.9um (segment mean of clear pixels)". The same applies to all CSR entries except for wv62 where it would be "TOA Brightness Temperature at 3.9um (segment mean of clear and low cloud pixels)"

Reply:
done